features-json/variable-fonts.json

```diff
@@ -170,8 +170,8 @@
       "63":"a #4",
       "64":"a #4",
       "65":"a #4",
-      "66":"a #4",
```

Comment:
This can change to `y` if there are no further issues. Else there should be a note describing why support is still partial.

Reply:
Done.